### PR TITLE
Add Labels on the cluster level

### PR DESCRIPTION
**Pull request by leangeder**
Allow gravitywell to set label on the cluster level.

### CODE REVIEW

File: configuration/configuration.go [hunk 1]
```diff
@@ -24,42 +25,43 @@ type ApplicationCluster struct {
...
+	InitialNodeType  string `yaml:"InitialNodeType"`
+	Labels           string `yaml:"Labels"`
+	Name             string `yaml:"Name"`
+	Project          string `yaml:"Project"`
+	NodePools        []struct {
+		NodePool struct {
+			Count    int    `yaml:"Count"`
+			Name     string `yaml:"Name"`
+			NodeType string `yaml:"NodeType"`
+			Labels   string `yaml:"Labels"`
```
Comment (Marvalero) [inline configuration/configuration.go]: Why not:
```
Labels map[string]string `yaml:"labels"`
```

The code would be simpler and the yaml nicer (same for the Labels in the Cluster)

**Comment by AlexsJones** [comment]
Please use GoLand IDE and disable reindentation of the entire file @leangeder

**Comment by AlexsJones** [comment]
Please resolve conflicts and run `go fmt ./...`

File: configuration/configuration.go (hunk 1 above)
Comment (leangeder) [reply configuration/configuration.go]: I have just done a copy of the code wrote by Alex

**Comment by leangeder** [comment]
@AlexsJones, I have solve the issue. Concern  GoLand IDE,  I am not a big fan of this IDE. Can you told me, what is the issue with the reindentation? I will fix my vim configuration.

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**